doc/reference/trd104-syscalls.md

```diff
@@ -496,14 +496,19 @@ failure variant of `Failure`, with an associated error code of
 handle userspace/kernel mismatches should be able to handle `Failure` in
 addition to the expected failure variant (if different than `Failure`).
 
-4.3.1 Command Identifier 0
---------------------------------
 
-Every device driver MUST implement command number 0 as the
-"exists" command.  This command always returns `Success`. This command
-allows userspace to determine if a particular system call driver is
-installed; if it is, the command returns `Success`. If it is not, the
-kernel returns `Failure` with an error code of `NODEVICE`.
+4.3.1 Command Idenfitier 0
+--------------------------
+
+ - [Added August, 2023](../rfcs/2023-08-18--CommandZeroSemantics.md).
+
+Command Identifier 0 is implemented by the core kernel and provides an
```

Comment:
whether or not it is provided by the core kernel seems out of scope of this TRD.

```suggestion
Command Identifier 0 provides an
```

Reply:
Updated